spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/domain/Image.java

```diff
@@ -0,0 +1,26 @@
+package io.crate.spring.jdbc.samples.domain;
+
+import java.util.Date;
+
+public class Image {
+
+    private String digest;
+
+    private Date last_modified;
```

Comment:
```suggestion
    private Date lastModified;
```

Reply:
Was necessary to have it serialized correctly